en/mkdocs.yml

```diff
@@ -550,9 +550,6 @@ nav:
           - ELK Based Analytics:
               - Setup: mi-analytics/mi-elk-installation-guide.md
               - Access the Dashboard: mi-analytics/mi-elk-dashboards.md
-          - MI Analytics:
-              # - Setup: mi-analytics/setting-up-mi-analytics.md
-              - Access the Dashboard: mi-analytics/using-the-analytics-dashboard.md
```

Comment:
Let's add redirects to Analytics overview page

Reply:
Fixed with [2133075](https://github.com/wso2/docs-mi/pull/948/commits/21330756e327fd8b4780e1b520d61a6edaf4b221)